src/components/launcherdbus.h

```diff
@@ -48,7 +51,7 @@ public slots:
     void showUpdatingProgress(QString packageName);
 
 private:
-    LauncherModel *m_model;
+    QList<LauncherModel *> m_models;
```

Comment:
Not really, both resolve to an array of LauncherModel * so it's much for muchness unless you want to get into discussions about the relative benefits of less generated code (QList) versus greater potential for inlining (QVector).